### ISSUE TITLE
Content: Neumitgliederinfos

### ISSUE THREAD

**Issue by xwolfde** [Piratenpartei/WebsiteRelaunch2017#8]
Unter den letzten BuVo wurde die Neumitgliederinfo sehr gut neu aufgelegt.
Dies sollten wir auch in der Website nutzen. 
Kann oder sollte man hieraus einen eigenständigen Bestandteil der Website machen?

**Comment by patrickschiffer** [comment]
Ja, aber erst mal niedrig priorisieren. Ich fände es gut, wenn es dort Downloads, HowTos/FAQs und eine Piratencloud geben würde, aber das ist wohl noch Zukunftsmusik :)

**Comment by patrickschiffer** [comment]
Ich hab unter "Mitglieder" https://www.piratenpartei.de/mitglieder/ mal das Mitgliederhandbuch verlinkt.
